quantmsrescore/snr.py

```
@@ -1,227 +1,297 @@
+
+    psm_file = output_dir / "psms.parquet"
+    search_param_file = output_dir / "search_params.parquet"
+    proteins_file = output_dir / "proteins.parquet"
+    protein_groups_file = output_dir / "protein_groups.parquet"
+
+    try:
+        out_path = Path(output)
+        pq.write_table(idparquet_psm, psm_file)
+        logger.info(f"psms.parquet file written to {out_path}")
+    except Exception as e:
+        logger.error(f"Failed to write psms.parquet psm file: {str(e)}")
```

Ruff (0.15.13)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)